structure/format.go

```diff
@@ -151,6 +152,12 @@ func (f *Formatter) outputFields(entry *Entry, raw json.RawMessage) {
 				continue
 			}
 			if !f.shouldSkipField(key, value) {
+				switch v := value.(type) {
+				case float64:
+					output = append(output, key+"="+strconv.FormatFloat(float64(v), 'f', -1, 64))
+				default:
+					output = append(output, fmt.Sprintf("%s=%v", key, value))
+				}
```

Comment:
The line below the switch there is still the old append, causing every field to be appended double.
```bash
$ echo '{"msg":"hi","x":876111112171}' | go run *.go
hi [x=8.76111112171e+11 x=876111112171]
```

I would remove the  original append line (line 161).

Reply:
my fault, just forgot run test suite before PR and remove this line :(